cmd/process-agent/collector.go

```diff
@@ -470,7 +470,7 @@ func (l *Collector) updateRTStatus(statuses []*model.CollectorStatus) {
 	maxInterval := 0 * time.Second
 	activeClients := int32(0)
 	for _, s := range statuses {
-		shouldEnableRT = shouldEnableRT || (s.ActiveClients > 0 && l.cfg.AllowRealTime)
+		shouldEnableRT = shouldEnableRT || s.ActiveClients > 0
```

Comment:
Could you put `AllowRealTime` back, the plan was to expose this in config separately.

Reply:
Oh sorry! Thank you for that

Reply:
Added!